packages/py-api/py_api/controllers/logs_controller.py

```diff
@@ -0,0 +1,30 @@
+from os.path import exists
+from pathlib import Path
+from typing import Final
+
+from fastapi.responses import FileResponse, JSONResponse
+from py_api.environment import IS_OFFLINE
+
+
+class LogsController:
+    LOGFILE_PATH: Final = f"{Path(__file__).parent.resolve().parent}/shared/logfile.log"
+
+    @classmethod
+    def get_log_file(cls) -> JSONResponse | FileResponse:
+
+        if IS_OFFLINE:
+            content = {
+                "message": "Log files are not available in local environments.",

```

Comment:
remove the dot at the end 😆 I'm sorry for making you do this but let's stick to doing the same for all of our response. Have dots only if the messages are more than one sentence.